backend/graph/api/model/status.go

```diff
@@ -0,0 +1,12 @@
+package model
+
+type canBeUnlisted interface {
+	Unlisted() bool
```

Comment:
Isnt this relying on an implementation detail? this mapping function is only to map statuses from graphql.
No need to make it too generic, it hurts readability.

Reply:
@fredrikved

Reply:
It doesn't matter too much. Super easy to change. I find this better than referencing common.Status. 

If we need more statuses later, we just reimplement this in another way.

Reply:
Yeah but in general if you do this everywhere it can get messy. something to consider